.github/workflows/static_analysis.yml

```diff
@@ -21,6 +22,14 @@ jobs:
         with:
           python-version: '3.7'
           cache: 'pip' #Cache dependencies
```

Comment:
add a space here?

Reply:
add a space above the with? To be honest the with is part of setup python action so it makes sense to be 'connected' like in the examples https://github.com/actions/setup-python